api/models/timelog.py

```diff
@@ -95,11 +95,11 @@ class Template(Base):
 
     @hybrid_property
     def start_time(self):
-        return int_to_time_string(self.init) if self.init else None
+        return int_to_time_string(self.init) if self.init is not None else None
```

Comment:
@dmtrek14 sorry for asking you to remove this one, when checking `if self.init` it considers 0 as false, so it wasn't parsing it.